fe2-android/README.md

```diff
@@ -0,0 +1,67 @@
+# student21_pop: fe2-android
+Proof-of-personhood, spring 2021: Android native front-end
+
+## Table of contents
+* [Technicalities](#technicalities)
+* [Setup](#setup)
+* [Github Actions](#github-actions)
+* [Coding Standards](#coding-standards)
+
+### Note
+
+This repository is being refactored and some classes and interfaces are due for removal.
```

Comment:
If possible, remove them before merging.
Nothing is more confusing than a code base with lots of dead code.